graphs/RuntimeMiddleware.fbp

```diff
@@ -0,0 +1,17 @@
+INPORT=Dispatch.IN:IN
+OUTPORT=Passed.OUT:PASS
+OUTPORT=NewActions.OUT:NEW
+
+'context:edges' -> ROUTES Dispatch(ui/DispatchAction)
+
+# Send selected edges to runtime
+'runtime,graphs[0]' -> KEYS GetEdgeState(ui/GetActionValues)
+Dispatch HANDLE[0] -> IN GetEdgeState
+GetEdgeState VALUES[0] -> RUNTIME SendEdges(runtime/SendEdges)
```

Comment:
Would prefer to use a JS array or object for such things, and then another component for picking out the particular element if/when needed. Don't like arrayports, and they are very badly supported in Flowhub.